apds9960/device.py

```diff
@@ -23,23 +23,23 @@ def __init__(self, bus, address=APDS9960_I2C_ADDR, valid_id=APDS9960_DEV_ID):
         # instance variables for gesture detection
         self.gesture_ud_delta_ = 0
         self.gesture_lr_delta_ = 0
-
+	
```

Comment:
Could you please remove those trailing whitespaces on empty lines? I'm fine with the other changes :-)

Reply:
They should be gone now 👍